submissions/AbdulkareemIdayatOlaide/javascriptTask/productpage.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>productPage</title>
+        <link href="index.css" rel="stylesheet">
+    </head>
+    <body>
+        <h1>product list</h1>
+        <button id="btn">search for product</button>
+        <br/>
+        <br/>
+        <div class="product-list" id="js-product-list"></div>
+        <br/>
+        <div class="product-box">
+           <div class="product-content">
+                <div class="product-image">
+                    <img src="decoration.png" class="images">
```

Comment:
this image is going to show on the web page because it is not referencing the right folder. change to ` <img src="pageImage/decoration.png" class="images">`